Client-Setup-Windows.md

```diff
@@ -1,4 +1,4 @@
-**Note: To be able to update the Final Fantasy XI client - and connect to private servers using the most recent version - you must be capable of connecting to retail at least once. Possible methods for doing so may be found below in [Section 2 - Connect to Retail](https://github.com/LandSandBoat/server/wiki/Client-setup-%5BWindows%5D#2-connecting-to-retail). LandSandBoat does not condone methods enabling client updates without having connected to SquareEnix's retail servers.**
+**Note: To be able to update the Final Fantasy XI client - and connect to private servers using the most recent version - you must be capable of connecting to retail at least once. Possible methods for doing so may be found below in [Section 2 - Connect to Retail](Client-Setup-Windows#2-connecting-to-retail). LandSandBoat does not condone methods enabling client updates without having connected to SquareEnix's retail servers.**
```

Comment:
needs the file extension looks like. I just tried it. adding the `.md` made it work as expected.

Reply:
the actual live wiki doesn't seem to care, in other locations copies of it do. Odd behavior on Gollum's part. Unsure how much we care right now, if we were to ever migrate off github we certainly would though.